rest/handler/loghandler.go

```diff
@@ -81,6 +87,12 @@ func (w *DetailLoggedResponseWriter) WriteHeader(code int) {
 	w.writer.WriteHeader(code)
 }
 
+func (w *DetailLoggedResponseWriter) Flush() {
+	if flusher, ok := http.ResponseWriter(w.writer).(http.Flusher); ok {
```

Comment:
`if flusher, ok := w.writer.(http.Flusher); ok {`

I think this also works.

Reply:
`invalid type assertion: w.writer.(http.Flusher) (non-interface type *LoggedResponseWriter on left)`
it raise error like above.